src/sv_move.c

```diff
@@ -313,7 +313,7 @@ void SV_NewChaseDir (edict_t *actor, edict_t *enemy, float dist)
 	}
 
 	// try other directions
-	if ( ((rand()&3) & 1) ||  abs(deltay)>abs(deltax))
+	if ( ((rand()&3) & 1) ||  fabs(deltay)>fabs(deltax))
```

Comment:
Just curious.. why `fabs()` instead of `fabsf()`? `fabs()` assumes input is a double, but `deltay` and `deltax` are both float. I guess we don't really care about the amount of precision since 24 vs 52 bits isn't going to make a huge difference?

Reply:
Weird, when I checked earlier I had the impression `float` was the default but double checking it is indeed `double`. 
We can switch it to `fabsf` but maybe we should do it to in the `PF_fabs(void)` of `pr_cmds.c` as well to be consistent?

Reply:
@deurk : `PF_fabs(void)` still uses `fabs()`. I think for correctness we should change line 316 to: `if ( ((rand()&3) & 1) ||  fabsf(deltay)>fabsf(deltax))` does that seem reasonable?